src/app/views/admissions/student-form/student-utils/ProfileForm.jsx

```diff
@@ -1,16 +1,16 @@
 import React, { useState } from 'react';
-import { Formik, yupToFormErrors } from 'formik';
-import {
-  Grid,
-  TextField,
-  Button,
-} from '@material-ui/core';
-
+import { Formik } from 'formik';
+import { Grid, TextField, Button } from '@material-ui/core';
 import { useHistory } from 'react-router-dom';
-import bc from 'app/services/breathecode';
+import PropTypes from 'prop-types';
+import bc from '../../../../services/breathecode';
 import axios from '../../../../../axios';
 import { AsyncAutocomplete } from '../../../../components/Autocomplete';
 
+const propTypes = {
+  initialValues: PropTypes.string.isRequired,
```

Comment:
`initialValues` probably is an object change ~~`string`~~ to `object`

expected:
```js
initialValues: PropTypes.objectOf(PropTypes.object).isRequired,
```